src/components/RestUsers/EditUserForm.js

```diff
@@ -1,7 +1,82 @@
 import React from 'react';
+import { useFormWithErrors, useRestfulFetch } from '../../hooks';
+import { client } from '../../utils/api';
+import schema from '../../validation/EditUserSchema';
+import 'antd/dist/antd.css';
+import { Button, Form, Select } from 'antd';
 
 const EditUserForm = props => {
-  return <>Edit Existing User</>;
+  const { data } = useRestfulFetch(props.href);
+  const { values, disabled, onChange, setValues } = useFormWithErrors(
+    schema,
+    data,
+    false
+  );
+
+  React.useEffect(() => {
+    if (data) {
+      setValues(prevValues => ({ ...prevValues, ...data }));
+    }
+  }, [data, setValues]);
+
+  const changeValues = e => {
```

Comment:
Use the same code to replace this `changeValues` as well

Reply:
great catch @BUGBOUNTYchrisg8691.

Reply:
Also implemented here.